src/main/java/smokefree/projection/InitiativeProjection.java

```diff
@@ -145,4 +160,9 @@ public Progress progress() {
         return progress;
     }
 
+
+    public Map<String, Playground> playgrounds() {
```

Comment:
I don't like this style. Can you make it something like getAllPlaygrounds() ? 
Also I would just return the values

Reply:
made changes as suggested.